### ISSUE TITLE
camerad open bug

### ISSUE THREAD

**Issue by astronomerdave** [CaltechOpticalObservatories/NGPS#107]
Not sure how, but camerad got into a state where `open 0` reported that `0` was not a valid device. This was a result of an internal call to `set_image_size`. Checking with the `imsize` command reported

```
  Specify <chan> from {   I R }
       or <dev#> from { 0 0 1 }

```
so it seems that an extra device "0" got included without an accompanying channel. That's certainly the source of the error but unclear how it got into this state.

**Comment by astronomerdave** [comment]
fixed with commit f73e00fbf5a85d37ef85175eb18ae246a7935ecb

**Comment by astronomerdave** [comment]
Commit f73e00fbf5a85d37ef85175eb18ae246a7935ecb introduced a new bug, now fixed in 0923503d89d14ade4f5d6f9e8b17e1a532047f44
